toolkit/tools/imagecustomizer/README.md

```diff
@@ -67,6 +67,9 @@ Disadvantages:
    For a description of all the command line options, see:
    [Mariner Image Customizer command line](./docs/cli.md)
 
+   For more information about creating ISO images,
+   see [Mariner Image Customizer ISO Support](./docs/iso.md)
```

Comment:
Should this be linked from the configuration.md as well, as it contains extension of the API?